done.md

```diff
@@ -0,0 +1,6 @@
+You're done, congrats!   
+
+Now that you've skilled up on the basics, go ahead and:    
+* [Get started](https://github.com/KILTprotocol/sdk-js/blob/develop/docs/getting-started.md) with our hands-on intro to KILT's more advanced concepts; 
```

Comment:
Nice :) 

Should it be advanced concepts or developing your own applications on top of the KILT Protocol? 

I would maybe add the API as well.

Reply:
Great ideas, included now 👍